kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtWireServiceImpl.java

```diff
@@ -71,22 +71,21 @@
  */
 public final class GwtWireServiceImpl extends OsgiRemoteServiceServlet implements GwtWireService {
 
+    private static final int SERVICE_WAIT_TIMEOUT = 60000;
```

Comment:
Is there any specific reason of setting it to 1 min? For embedded devices, I don't think it's good idea to wait for this much amount of time.